ECE tools 2002.1.5 Release (#8578)
magento/devdocs: src/cloud/env/variables-build.md

    SCD_COMPRESSION_TIMEOUT: 800
```

### `SCD_NO_PARENT`

-  **Default**—`false`

> -  **Version**—Magento 2.3.0 and later

Reviewer: Here should be 2.4.2 and later, firstly it was 2.3.0 but it was fixed in the code of ece-tools

Set to `true` to prevent generating static content for parent themes during the build phase.

> On the build phase, we recommend setting `SCD_NO_PARENT: false` so that static content for the parent themes is generated during the build phase where the process will not impact site deployment or cause unnecessary site downtime. See [Static content deployment]({{site.baseurl}}/cloud/deploy/static-content-deployment.html).

Reviewer 2: Historically on SKIP_SCD, this sentence has been edited many times. I agree that the original was causing confusion, but this version notes the build phase twice: on and during. So, I see the same sentence is now used for this variable and still seems a little confusing. Maybe this suggestion works?

```suggestion
We recommend setting `SCD_NO_PARENT: false` during the build phase so that generating static content for the parent themes does not impact site deployment or cause unnecessary site downtime. See [Static content deployment]({{site.baseurl}}/cloud/deploy/static-content-deployment.html).
```
